test/integration/test_payroll_process_scenarios/classic.js

```diff
@@ -0,0 +1,567 @@
+       id : 7,
+       currency_id : 1,
+       rate : 32.5,
+       tranche_annuelle_debut : 10260000,
+       tranche_annuelle_fin : 13908000,
+       tranche_mensuelle_debut : 855000,
+       tranche_mensuelle_fin : 1159000,
+       ecart_annuel : 3648000,
+       ecart_mensuel : 304000,
+       impot_annuel : 1185600,
+       impot_mensuel : 98800,
+       cumul_annuel : 3567580,
+       cumul_mensuel : 297298,
+       taxe_ipr_id : 1,
+     },
+     {
+       id : 8,
+       currency_id : 1,
+       rate : 35,
+       tranche_annuelle_debut : 13908000,
+       tranche_annuelle_fin : 16824000,
+       tranche_mensuelle_debut : 1159000,
+       tranche_mensuelle_fin : 1402000,
+       ecart_annuel : 2916000,
+       ecart_mensuel : 243000,
+       impot_annuel : 1020600,
+       impot_mensuel : 85050,
+       cumul_annuel : 4588180,
+       cumul_mensuel : 382348,
+       taxe_ipr_id : 1,
+     },
+     {
+       id : 9,
+       currency_id : 1,
+       rate : 37.5,
+       tranche_annuelle_debut : 16824000,
+       tranche_annuelle_fin : 22956000,
+       tranche_mensuelle_debut : 1402000,
+       tranche_mensuelle_fin : 1913000,
+       ecart_annuel : 6132000,
+       ecart_mensuel : 511000,
+       impot_annuel : 2299500,
+       impot_mensuel : 191625,
+       cumul_annuel : 6887680,
+       cumul_mensuel : 573973,
+       taxe_ipr_id : 1,
+     },
+     {
+       id : 10,
+       currency_id : 1,
+       rate : 40,
+       tranche_annuelle_debut : 22956000,
+       tranche_annuelle_fin : 100000000000000,
+       tranche_mensuelle_debut : 1913000,
+       tranche_mensuelle_fin : 1913000,
+       ecart_annuel : 0,
+       ecart_mensuel : 0,
+       impot_annuel : 0,
+       impot_mensuel : 0,
+       cumul_annuel : 6887680,
+       cumul_mensuel : 573973,
+       taxe_ipr_id : 1,
+     }],
+    },
+  };
+
+  const paramsConfig = {
+    currency : '2',
+    employees : '0D68B9D64D1F45EDA43302D619F7CE98',
+    idPeriod : params.periodPaie,
+    lang : 'fr',
+    payslip : 'true',
+    renderer : 'json',
+  };
+
+  const paramsMultiConfig = {
+    currency : 2,
+    employees : [
+      'B8C3674B2F264B598C16FBA76F5116D1',
+      '51D6A9297B464327B6EB8155BDBDC9E0',
+    ],
+    idPeriod : params.periodPaie,
+    lang : 'fr',
+    payslip : 'true',
+    renderer : 'json',
+  };
+
+  // Amelia Rose Thornton
+  it('POST /multiple_payroll/:id/ Payroll Management: Continuity of Work For Employees Amelia Rose Thornton', () => {
+    return agent.post('/multiple_payroll/'.concat(params.periodPaie, '/configuration'))
+      .send(dataConfiguration)
+      .then((res) => {
+        expect(res).to.have.status(201);
+
+        return agent.get('/reports/payroll/payslip')
+          .query(paramsConfig);
+      })
+      .then((res) => {
+        // The seniority bonus is calculated automatically by the system, using the formula:
+        // Base Salary × Seniority Rate × Years of Service
+
+        expect(res.body.rubrics[2].total).to.equal(330.75);
```

Comment:
Instead of using array notation here  (e.g. `res.body.rubrics[2]`), can you call these rubrics by their name/abbr?   This test feels very brittle.

It may mean you'll need to reformat the way the ReportManager presents it's data, but that is probably a good thing for maintainability.

Reply:
Done